source/bayernfahrplan/fahrplanparser/json/jsonUtils.d

```diff
@@ -0,0 +1,222 @@
+module bayernfahrplan.fahrplanparser.json.jsonutils;
+
+import std.json : JSONValue;
+import std.datetime : DateTime, Date, TimeOfDay;
+import std.conv : to;
+import fluent.asserts : should;
+
+public:
+// Todo: Implement proper error handling
+string getLine(ref JSONValue departureInfo)
+{
+    import std.json : JSON_TYPE;
+
+    auto lineNumber = departureInfo["mode"]["number"];
+    switch (lineNumber.type) with (JSON_TYPE)
+    {
+    case STRING:
+        return lineNumber.str;
+    case INTEGER:
+        return lineNumber.integer.to!string;
+    default:
+        return ""; // TODO: Create UnexpectedJsonType or the like.
+    }
+}
+
+@system
+{
+    import std.stdio : writeln;
+
+    unittest
+    {
+        auto classUnderTest = JSONValue(["mode" : ["number" : "1"]]);
```

Comment:
Again, another name than `classUnderTest` should be used. Also applies to the following tests.

Maybe it does not even need a name:
```D
JSONValue(["mode" : ["number" : "1"]]).getLine.should.equal("1");
```
That would be ok, too, imo.

Reply:
Hm, I'm a fan of separation of test data and actual test execution, as it does not create to much overhead and is easier adaptable in case. But I will rename it.